exampleSite/content/post/2020-11-10-r-markdown-demo/index.Rmd

```diff
@@ -1,9 +1,9 @@
 ---
-title: R Markdown Demo
-author: Yihui Xie
+title: "R Markdown Demo"
+author: "Yihui Xie"
 date: '2020-11-10'
 slug: r-markdown-demo
-features: [+sticky_menu]
+features: +sticky_menu
```

Comment:
This needs to be an array at the moment (there are other features that could be enabled), but I can probably add a workaround to allow scalars.

Reply:
I didn't change that- whenever I knit, it changes 😢

Reply:
Aha, it doesn't happen if I click the knit button, but because the output format isn't in the dropdown?? when I click "knit to html" it removes the brackets.

![Kapture 2020-11-13 at 12 07 28](https://user-images.githubusercontent.com/12160301/99099769-e87a6180-25a8-11eb-8b9f-e4012aefe4b3.gif)